src/trivyHelper.ts

```diff
@@ -162,6 +162,21 @@ export function getSeveritiesToInclude(warnIfInvalid?: boolean): string[] {
     return severities;
 }
 
+export function getTrivyFilteredOutput(): any {
```

Comment:
Trivy is redundant in the name.
Suggestion: `getFilteredOutput`. Or `getScanOutput` may be a better name?

Reply:
changed to getFilteredOutput